DigitalLearningSolutions.Web/Views/SuperAdmin/Shared/_SideNavMenu.cshtml

```diff
@@ -3,28 +3,24 @@
 
 <h2 class="side-nav__heading">System</h2>
 <ul class="nhsuk-list side-nav__list">
-  <vc:side-menu-link asp-action="Index"
-                     asp-controller="Faqs"
-                     link-text="Configuration"
-                     is-current-page="@(Model == SuperAdminSystemPage.Configuration)" />
+  <vc:side-menu-link-with-href href=""
+                               link-text="Configuration"
+                               is-current-page="@(Model == SuperAdminSystemPage.Configuration)" />
 
-  <vc:side-menu-link asp-action="Index"
-                     asp-controller="Faqs"
-                     link-text="Manage Lists"
-                     is-current-page="@(Model == SuperAdminSystemPage.ManageLists)" />
+  <vc:side-menu-link-with-href href=""
+                               link-text="Manage Lists"
+                               is-current-page="@(Model == SuperAdminSystemPage.ManageLists)" />
 
-  <vc:side-menu-link asp-action="Index"
-                     asp-controller="Faqs"
-                     link-text="Notifications"
-                     is-current-page="@(Model == SuperAdminSystemPage.Notifications)" />
+  <vc:side-menu-link-with-href href=""
+                               link-text="Notifications"
+                               is-current-page="@(Model == SuperAdminSystemPage.Notifications)" />
 
-  <vc:side-menu-link asp-action="Index"
-                     asp-controller="Faqs"
-                     link-text="Resources"
-                     is-current-page="@(Model == SuperAdminSystemPage.Resources)" />
+  <vc:side-menu-link-with-href href=""

```

Comment:
Why the view component change? I'm not sure we use `vc:side-menu-link-with-href` anywhere else, likely it was deprecated but not removed.

Reply:
just so the links don't go anywhere while we dont have the other pages